src/App.js

```diff
@@ -66,10 +82,13 @@ export class App extends Component{
   }
 
   render() {
+    if (this.state.isLoading) {
+      return (<h3>Loading...</h3>);
+    }
     if (this.state.numPlayers > 1 && !this.state.word && History[History.length -1] !== '/word-selector') {
       window.setTimeout(() => History.push('/word-selector'), 1);
     } else if (this.state.numPlayers > 1 && this.state.word && History[History.length -1] !== '/gamepage') {
-      
+
```

Comment:
Good job removing tech debt. We have to do these things, otherwise Big Tech wins.

Reply:
ONE HUNDRED FUNNIES TO YOU